app/views/roles/edit.html.erb

```diff
@@ -1,26 +1,8 @@
 <h2>Role:</h2>
-<%= bootstrap_form_for @role, :url=>role_management.role_path(@role) do |f| %>
   <%= f.text_field :name, :label=> "Role Name" %>
-
-    <%= f.submit "Update" %>
-
-<% end %>
-<% if can? :destroy, Role %>
-  <%= button_to "Delete", role_management.role_path(@role), :method=>:delete, :class=>'btn btn-danger' %>
-<% end %>
 <h3>Accounts</h3>
 <ul>
   <% @role.users.each do |user| %>
-    <li id=<%= user.user_key %>><%= user.user_key %>
-      <% if can? :remove_user, Role %>
-        <%= button_to "Remove User", role_management.role_user_path(@role, user.id), :method=>:delete, :class=>'btn btn-danger' %>
-      <% end %>
-    </li>
+    <li id=<%= user.user_key %>><%= user.user_key %></li>
   <% end %>
 </ul>
-<h3>Add New Account</h3>
-<%= bootstrap_form_tag :url=> role_management.role_users_path(@role) do |f| %>
-  <%= f.text_field 'user_key', :label=>"User" %>
-  <%= f.submit "Add" %>
-  <%= link_to "Cancel", role_management.roles_path, :class => 'btn btn-default' %>
-<% end %>
```

Comment:
Why changes to this view?

Reply:
It's been so long since I submitted the old PR, but if I remember right it was to stop people from adding or updating roles for themselves. Once this is in we should only be editing a few files. Might be overkill, but it doesn't hurt anything either.